react/scss/person-graph.scss

```diff
@@ -0,0 +1,19 @@
+@import 'common';
+
+.link line {
+  stroke: #999;
+  stroke-opacity: .6;
+}
+
+.node--group {
+  cursor: pointer;
+}
+
+.node--leaf {
```

Comment:
I'm not sure either, but looks like a modifier / state to me more than a new element, because it's still a node.